bootstrap/helpers/docker.php

```
@@ -1272,3 +1272,31 @@ function generateDockerEnvFlags($variables): string
+
+/**
+ * Auto-inject -f and --env-file flags into a docker compose command if not already present
+ *
+ * @param  string  $command  The docker compose command to modify
+ * @param  string  $composeFilePath  The path to the compose file
+ * @param  string  $envFilePath  The path to the .env file
+ * @return string The modified command with injected flags
+ */
+function injectDockerComposeFlags(string $command, string $composeFilePath, string $envFilePath): string
+{
+    $dockerComposeReplacement = 'docker compose';
```

PHPMD (2.15.0)

Avoid excessively long variable names like $dockerComposeReplacement. Keep variable name length under 20. (undefined)

(LongVariable)